README.md

```diff
@@ -31,6 +31,15 @@ self.volumeButtonHandler = [JPSVolumeButtonHandler volumeButtonHandlerWithUpBloc
 }];
 ```
 
+To enbable/disable the handler:
+```objective-c
+	// Start
+	[self.volumeButtonHandler startHandler:YES]; 
```

Comment:
So change the line 
// Start
to 
// Start : BOOL specify whether you would like to disableDefaultVolumeHandler